Mail/Utils/Notification+Name.swift

```diff
@@ -20,6 +20,7 @@ import Foundation
 
 public extension Notification.Name {
     static let onUserTappedNotification = Notification.Name("userTappedNotification")
+    static let onUserTappedReplyToNotification = Notification.Name("userTappedReplyToNotification")
```

Comment:
I agree we should decide on a pattern and stick to it. Now should it include `Notification` or not is something I leave it to you to decide.

Reply:
I have a slight preference for the name without Notification because it's already a Notification.Name, but that's subjective.

Reply:
Yes but here the issue is that the `Notification` in `userTappedReplyToNotification` is a push notification. The Notification type is an obj-c notification. So it might be more confusing to have a `Notification.Name.userTapped`

Reply:
I am just talking about this part `Notification.Name("userTappedNotification")` 😊